render/expected/expected.go

```diff
@@ -17,18 +17,21 @@ var (
 		ID:                unknownPseudoNode1ID,
 		LabelMajor:        "10.10.10.10",
 		Pseudo:            true,
+		NodeMetadata:      report.MakeNodeMetadata(map[string]string{}),
```

Comment:
The problem is that the expression

```go
report.NodeMetadata{}
```

used to produce a usable object, and now it doesn't. So I updated all instances of that to

```go
report.MakeNodeMetadata(map[string]string{})
```

which in turn caused test failures, as some expected NodeMetadata fields changed from having a nil Metadata member to a constructed-but-empty Metadata member. Does it make sense?